ga-ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldGAspec.json

```diff
@@ -1580,6 +1580,14 @@
       }
     ]
   },
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
```

Comment:
could we please double check if only systemupdate role is required for this field?

Reply:
this file is not inside civ-6011 change list

Reply:
this change comes from https://github.com/hmcts/civil-general-apps-ccd-definition/pull/239/files